examples/messaging/iterating-multiple-subscriptions/java/Main.java

```diff
@@ -0,0 +1,52 @@
+package example;
+
+import io.nats.client.Connection;
+import io.nats.client.Nats;
+import io.nats.client.Dispatcher;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+
+public class Main {
+    public static void main(String[] args) {
+        String natsURL = System.getenv("NATS_URL");
+        if (natsURL == null) {
+            natsURL = "nats://127.0.0.1:4222";
+        }
+
+        // Initialize a connection to the server. The connection is AutoCloseable
+        // on exit.
```

Comment:
You can remove this, it's not in any other example.

Reply:
When looking at the other available examples, Rust & .NET v2, it's indeed not there.
But for the Java examples for both pub/sub and request/reply it is there.

Would you like me to remove this for these new Java examples, or keep it in for consistency in the Java examples?